pom.xml

```diff
@@ -270,8 +270,8 @@
   </profiles>
 
   <properties>
-    <frontend.version>v3.1.5</frontend.version>
-    <frontend.groupId>com.github.cbioportal</frontend.groupId>
+    <frontend.version>2b16f8bdfa6c862add35397ece92141dc6da844d</frontend.version>
+    <frontend.groupId>com.github.inodb</frontend.groupId>
```

Comment:
will update this to frontend repo once https://github.com/cBioPortal/cbioportal-frontend/pull/2896 is merged